### PR TITLE
Fixed remove_monitoring_cluster script to correctly remove v4m-kubelet service

### PR DESCRIPTION
**Pull request by cumcke**
I forgot to include the Openshift in the previous CHANGELOG so I'm adding it in this pull request.

### CODE REVIEW

File: monitoring/bin/remove_monitoring_cluster.sh [hunk 1]
```diff
@@ -64,6 +64,9 @@ log_verbose "Removing configmaps and secrets"
 kubectl delete cm --ignore-not-found -n $MON_NS -l sas.com/monitoring-base=kube-viya-monitoring
 kubectl delete secret --ignore-not-found -n $MON_NS -l sas.com/monitoring-base=kube-viya-monitoring
 
+log_verbose "Removing Kubelet"
```
Comment (gsmith-sas) [inline monitoring/bin/remove_monitoring_cluster.sh:67]: According to the Kubernetes doc "The kubelet is the primary "node agent" that runs on each node."  Given that, this message migh alarm some folks.  I suggest updating the message to "Removing kubelet service".

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,15 @@
 # SAS Viya Monitoring for Kubernetes
 
+## UNRELEASE
+
+* **Overall**
+
+* **Metrics**
+  * [FIX] - Deploy_monitoring_cluster script now correctly removes the v4m-kubelet service that is deployed by deploy_monitoring_cluster.
```
Comment (gsmith-sas) [inline CHANGELOG.md:8]: A little too verbose...and incorrect (it is the remove_monitoring_cluster script that does the removing).  Maybe something like  "[FIX] - Remove v4m-kubelet service when removing cluster-level metric monitoring components"

**Review by gsmith-sas — CHANGES_REQUESTED** [review]
A couple of tweaks/corrections to the wording of things.

**Review by gsmith-sas — APPROVED** [review]
LGTM

**Review by ceelias — APPROVED** [review]

**Pull request merged**